src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -300,8 +288,9 @@ of the [`IdealGlmMhdEquations2D`](@ref).
 
 This implementation uses a non-conservative term that can be written as the product
 of local and symmetric parts. It is equivalent to the non-conservative flux of Bohm
-et al. (`flux_nonconservative_powell`) for conforming meshes but it yields different
-results on non-conforming meshes(!).
+et al. [`flux_nonconservative_powell`](@ref) for conforming meshes but it yields different
+results on non-conforming meshes(!). On curvilinear meshes this formulation applies the
+local normal direction compared to the averaged one used in [`flux_nonconservative_powell`](@ref).
```

Comment:
This would also be true in 3D but the subcell limiting implementation is not done (yet) right @amrueda ? That is why a corresponding comment in the 3D ideal GLM-MHD file is not done?

Reply:
Sorry, I just saw this... That is correct! The comment would also apply in 3D, but the flux `flux_nonconservative_powell_local_symmetric` has not been implemented for the 3D equations.